cdap-cli/src/main/java/co/cask/cdap/cli/CLIConfig.java

```diff
@@ -201,7 +200,7 @@ private boolean saveAccessToken(AccessToken accessToken, String hostname) {
     File accessTokenFile = getAccessTokenFile(hostname);
 
     try {
-      if (accessTokenFile.createNewFile()) {
+      if (accessTokenFile.exists() || accessTokenFile.createNewFile()) {
```

Comment:
Sorry, deleted a previous comment which I realized didn't make sense after reading the API docs.